doc/api/process.markdown

```diff
@@ -679,24 +679,25 @@ for the source tarball and headers-only tarball.
 
 `process.release` contains the following properties:
 
-* `name`: a string with a value that will always be `"io.js"` for io.js.
+* `name`: a string with a value that will always be `"node"` for Node.js. For
+  legacy io.js releases, this will be `"io.js"`.
 * `sourceUrl`: a complete URL pointing to a _.tar.gz_ file containing the
   source of the current release.
 * `headersUrl`: a complete URL pointing to a _.tar.gz_ file containing only
   the header files for the current release. This file is significantly smaller
   than the full source file and can be used for compiling add-ons against
-  io.js.
-* `libUrl`: a complete URL pointing to an _iojs.lib_ file matching the
+  Node.js.
+* `libUrl`: a complete URL pointing to an _node.lib_ file matching the
   architecture and version of the current release. This file is used for
-  compiling add-ons against io.js. _This property is only present on Windows
-  builds of io.js and will be missing on all other platforms._
+  compiling add-ons against Node.js. _This property is only present on Windows
+  builds of Node.js and will be missing on all other platforms._
 
 e.g.
 
-    { name: 'io.js',
-      sourceUrl: 'https://iojs.org/download/release/v2.3.5/iojs-v2.3.5.tar.gz',
-      headersUrl: 'https://iojs.org/download/release/v2.3.5/iojs-v2.3.5-headers.tar.gz',
-      libUrl: 'https://iojs.org/download/release/v2.3.5/win-x64/iojs.lib' }
+    { name: 'node',
+      sourceUrl: 'https://nodejs.org/download/release/v4.0.0/node-v4.0.0.tar.gz',
```

Comment:
I think this is ok. It will, it has to for the first 4.0.0 dev release / 4.0.0 itself.